com.civicrm.hrjobroles/js/test/controllers/hr-job-roles-controller/fetching-dates-from-contract_test.js

```diff
@@ -0,0 +1,171 @@
+define([
+    'common/angular',
+    'mocks/job-roles',
+    'common/angularMocks',
+    'job-roles/app'
+], function (angular, Mock) {
+    'use strict';
+
+    describe('Fetching Dates from contract', function () {
+        var ctrl, scope;
+
+        beforeEach(module('hrjobroles'));
+        beforeEach(inject(function ($controller, $rootScope) {
+            scope = $rootScope.$new();
+            ctrl = $controller('HRJobRolesController', { $scope: scope, format: 'DD/MM/YYYY' });
+
+            ctrl.contractsData = angular.copy(Mock.contracts_data);
+        }));
+
+        describe('Checking if dates entered in job role are th same as those in contracts', function () {
+
+            it('should check if entered dates are custom', function () {
```

Comment:
The `it` descriptions should be in this format `it('checks if entered dates are custom')` not in the "should" format